tfjs-core/src/ops/conv2d_test.ts

```diff
@@ -228,6 +228,72 @@ describeWithFlags('conv2d', ALL_ENVS, () => {
     expectArraysClose(resultData, [20, 26, 13, 12]);
   });
 
+  it('x=[4,2,2] f=[1,1,4,4] s=1 d=1 p=same NCHW', async () => {
+    // Skip webgl backend due to bug
```

Comment:
LGTM. 
Please ensure this case has full coverage for: 
batchA==1&batchB==1,
batchA==1,
batchB==1,
Or should add some matmul case for this?

Reply:
There are already some ones in matmul test. Just make them tested for webgpu.